src/components/Contacts/ContactDetails/ContactDonationsTab/PartnershipInfo/EditPartnershipInfoModal/EditPartnershipInfoModal.tsx

```diff
@@ -349,11 +353,16 @@ export const EditPartnershipInfoModal: React.FC<
                       },
                     }}
                   >
-                    {Object.values(StatusEnum).map((value) => (
-                      <MenuItem key={value} value={value}>
-                        {getLocalizedContactStatus(t, value)}
-                      </MenuItem>
-                    ))}
+                    {phases?.map((phase) => [
+                      <ListSubheader key={phase?.name}>
+                        {phase?.name}
+                      </ListSubheader>,
+                      phase?.contactStatuses.map((s: StatusEnum) => (
```

Comment:
Type isn't needed, and we can use a more descriptive variable name.

```suggestion
                      phase?.contactStatuses.map((status) => (
```